cmd/root.go

```diff
@@ -129,3 +129,11 @@ Additional help topics:{{range .Commands}}{{if .IsAdditionalHelpTopicCommand}}
 
 Use "{{.CommandPath}} [command] --help" for more information about a command.{{end}}
 `
+
+// Return use name and alias based on whether we're embedded in a wash shell.
+func commandNames(name string) (string, []string) {
```

Comment:
Wonder if there's a more descriptive name that captures the "Use this method if your command conflicts with a (common) shell command/built-in" intent of this method.

Reply:
Probably. generateShellAlias?

Reply:
That works.